### PR TITLE
*[ios] refactor gradient color

### PR DESCRIPTION
**Pull request by acton393**
I make gradient color function as utility to use

### CODE REVIEW

File: ios/sdk/WeexSDK/Sources/Utility/WXBoxShadow.m [hunk 1]
```diff
@@ -11,10 +11,11 @@
 #import "WXUtility.h"
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Utility/WXBoxShadow.m]: Maybe  you  should  concentrate more on  this file change @kfeagle

**Comment by cxfeng1** [comment]
@acton393  Please resolve conflicts first,  @boboning  Please have a review on it.

File: ios/sdk/WeexSDK/Sources/Utility/WXUtility.h [hunk 2]
```diff
@@ -357,6 +357,33 @@ CGRect WXPixelFrameResize(CGRect value) DEPRECATED_MSG_ATTRIBUTE("Use WXPixelSca
 CGPoint WXPixelPointResize(CGPoint value) DEPRECATED_MSG_ATTRIBUTE("Use WXPixelScale Instead");
 + (UIFont  * _Nullable )fontWithSize:(CGFloat)size textWeight:(CGFloat)textWeight textStyle:(WXTextStyle)textStyle fontFamily:(NSString * _Nullable)fontFamily DEPRECATED_MSG_ATTRIBUTE("Use +[WXUtility fontWithSize:textWeight:textStyle:fontFamily:scaleFactor:]");
 
+
```
Comment (cxfeng1) [inline ios/sdk/WeexSDK/Sources/Utility/WXUtility.h]: WXUtility is too heavy now, maybe we should take another place

Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Utility/WXUtility.h]: sure, it has become too complicated

**Pull request merged**